src/jacobians.jl

```diff
@@ -28,16 +28,16 @@ function JacobianCache(
         _fx1 = copy(x)
     end
 
-    JacobianCache(x1,_fx,_fx1,fdtype,returntype,inplace;colorvec=colorvec,sparsity=sparsity)
```

Comment:
this is upstream breaking?

Reply:
Indeed, but it would be meaningless to keep the inplace option.